### ISSUE TITLE
Crash on first cave level, after secluded hut.

### ISSUE THREAD

**Issue by DDR0** [frogatto/frogatto#402]
```
CRITICAL: custom_object.cpp:804 ASSERTION FAILED: Object legacy_particle_attache is invalid, property customized_spawn_rate_modulations expected to be [{expression: function(obj legacy_particle_attache) -> int, particle_id: string}]|null but found [
        {
                "expression": "@eval delay_until_end_of_loading(q(bind_closure(def() lib.math.round((wave(cycle)+1000)/100 + 10),deserialize('a52ccb8143e2473fa062e26383b5fd2e'))))",
                "particle_id": "torch_flames"
        }
] which is of type [{expression: function() -> int, particle_id: string},] 0

CRITICAL: 
---
CRITICAL: stack trace:
CRITICAL:   ./anura : CustomObject::validate_properties()+0x4ac
CRITICAL:   ./anura : CustomObject::createObject()+0x3b
CRITICAL:   ./anura : ()+0xf05931
CRITICAL:   ./anura : CustomObject::executeCommand(variant const&)+0x178
CRITICAL:   ./anura : CustomObject::executeCommand(variant const&)+0x12f
CRITICAL:   ./anura : CustomObject::handleEventInternal(int, game_logic::FormulaCallable const*, bool)+0x273
CRITICAL:   ./anura : CustomObject::handleEvent(int, game_logic::FormulaCallable const*)+0x1d5
CRITICAL:   ./anura : CustomObject::createObject()+0x55
CRITICAL:   ./anura : Level::do_processing()+0x4a1
CRITICAL:   ./anura : Level::process()+0x14e
CRITICAL:   ./anura : LevelRunner::play_cycle()+0x1a5c
CRITICAL:   ./anura : LevelRunner::play_level()+0xda
CRITICAL:   ./anura : main()+0x63b1
CRITICAL:   /lib/x86_64-linux-gnu/libc.so.6 : __libc_start_main()+0xeb
CRITICAL:   ./anura : _start()+0x2a
CRITICAL: ---
INFO: asserts.cpp:85 : ATTEMPTING TO SEND CRASH REPORT...
[1]    12045 abort (core dumped)  ./anura
```

**Comment by DDR0** [comment]
CLI `./anura --level=rock-a-fort-cave.cfg` reproduces.

**Comment by DDR0** [comment]
The legacy_particle_attache should be kept and made to work.

**Comment by DDR0** [comment]
Fixed by commit a8b07c31e.
